code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
ok, well I tried browserify-shim and it doesn't seem to work. Perhaps we should ask whoever is most familiar with the build setup?